src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -37,8 +44,13 @@
 rp_name_type = CLIArgumentType(help='Name of the recovery point.', options_list=['--rp-name', '-r'])
 backup_management_type = CLIArgumentType(help=backup_management_type_help, arg_type=get_enum_type(allowed_backup_management_types), options_list=['--backup-management-type'])
 workload_type = CLIArgumentType(help=workload_type_help, arg_type=get_enum_type(allowed_workload_types), options_list=['--workload-type'])
-restore_mode_type = CLIArgumentType(help=restore_mode_help, arg_type=get_enum_type(['OriginalLocation', 'AlternateLocation']), options_list=['--restore-mode'])
+restore_mode_type = CLIArgumentType(help=restore_mode_help, arg_type=get_enum_type(['OriginalLocation', 'AlternateLocation', 'AlternateWorkloadRestore', 'OriginalWorkloadRestore']), options_list=['--restore-mode'])
```

Comment:
can we separate the restore modes for afs/VM and SQL/Hana. Don't want to display users a wrong set of options in case of an error